packages/web/src/app/doc-pages/about/tutorial/angular-tutorial/angular-tutorial.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { RouterLink } from '@angular/router';
+
+import { CodeViewerComponent } from 'src/app/shared/component-documentation/ceg/code-generator/code-viewer/code-viewer.component';
+
+@Component({
+  selector: 'app-angular-tutorial',
+  templateUrl: './angular-tutorial.component.html',
+  standalone: true,
+  imports: [CodeViewerComponent, RouterLink],
+})
+export class AngularTutorialComponent {
+  codeStep1 = `npm install @elvia/elvis`;
+  codeStep2 = `import '@elvia/elvis/css/elvis.min.css';
+import '@elvia/elvis/elvis.js';`;
+  codeStep3 = `<h1 class=”e-title-lg”>Elvia title</h1>
+<button class=”e-btn”>Elvia button</button>`;
```

Comment:
It seems like you're already aware, but these quotation marks also end up the example, giving the example double quotation marks for the classes:
![Skjermbilde 2024-07-26 kl  14 52 20](https://github.com/user-attachments/assets/0137833d-f658-4ef6-bd3e-367f711e8e14)